code/game/jobs/job/silicon.dm

```diff
@@ -6,12 +9,21 @@
 	total_positions = 0 // Not used for AI, see is_position_available below and modules/mob/living/silicon/ai/latejoin.dm
 	spawn_positions = 1
 	selection_color = "#3F823F"
-	supervisors = "your laws"
+	supervisors = "your Laws"

```

Comment:
Laws are a proper noun?

Reply:
I feel that the AI's Laws have enough magnitude in relation to what is expected of the role to warrant a capital L, yes.

Reply:
I agree about the idea that laws should be capitalised in regards to the importance in following them to the best of the player's abilities.